src/app/helpers/isAmpPath.test.js

```diff
@@ -1,13 +1,10 @@
 import isAmpPath from './isAmpPath';
 
 describe('isAmpPath', () => {
-  it('should return false when passed a non-article path', () => {
```

Comment:
I think this is responsible for Code Climate's complaint, I'm not sure there's really a true parallel now the code is evaluating a different variable.